### DISCUSSION TITLE
Redis auth connection using TLS, how to configure?

### DISCUSSION THREAD

**Discussion by edemilson** [vernemq/vernemq#2035] [General]
I see there is ssl option in postgresql and mongodb, but in redis?

**Comment by ioolkos** [comment]
@edemilson No support for TLS for Redis, so far.
You'd have to update the eredis dependency (https://github.com/Nordix/eredis/) and extend the vmq_diversity app code. 
PRs welcome, not too much of an effort probably.

---
:point_right: Thank you for supporting VerneMQ: https://github.com/sponsors/vernemq
:point_right: Using the binary VerneMQ packages commercially (.deb/.rpm/Docker) requires a paid subscription.

**Reply by edemilson** [reply]
@ioolkos thanks!
